File: Travelo.Model/TripItem.cs
using System;
using System.Collections.Generic;
using System.Text;

namespace Travelo.Model
{
    public partial class TripItem
    {
        public int Id { get; set; }
        public DateTime CheckIn { get; set; }
        public DateTime CheckOut { get; set; }
        public int PricePerPerson { get; set; }
        public int NightsStay { get; set; }
       
        public int TripId { get; set; }
        public bool Expired { get; set; }
        public int TotalPrice { get; set; }
        public string Dates { get; set; }
    }
}

File: travelo_agency/lib/models/tripItem.dart
import 'package:json_annotation/json_annotation.dart';

part 'tripItem.g.dart';

@JsonSerializable()
class TripItem {
  int? id;
  DateTime? checkIn;
  DateTime? checkOut;
  int? pricePerPerson;
  int? nightsStay;
  int? tripId;
  bool? expired;
  int? totalPrice;
  String? dates;

  TripItem();

  factory TripItem.fromJson(Map<String, dynamic> json) =>
      _$TripItemFromJson(json);
}
